chart/templates/nginx.yaml

```diff
@@ -37,6 +42,14 @@ data:
         }
         {{- end }}
 
+        {{- if .Values.passthrough.enabled | default false}}
+        {{- range .Values.passthrough.upstreams }}
+        upstream backend_{{ .name }} {
+            server ###ZARF_VAR_BASE_IP###.{{ .hostId }}:443;
+        }
+        {{- end }}
```

Comment:
Is there a reason this is a range? The passthrough ingress gateway should be a single service, so the passthrough backend here should also be a single IP. It's also potentially "guaranteed" to a specific IP similar to how we have admin/tenant hardcoded?